ross/results.py

```diff
@@ -1360,15 +1360,15 @@ def plot_magnitude_bokeh(self, dof, units="m", **kwargs):
             y_axis_label = "Amplitude (m)"
         elif units == "mic-pk-pk":
             mag = 2 * mag * 1e6
-            y_axis_label = "Amplitude $(\mu pk-pk)$"
+            y_axis_label = "Amplitude (μ pk-pk)"
 
         # bokeh plot - create a new plot
         mag_plot = figure(
             tools="pan, box_zoom, wheel_zoom, reset, save",
-            width=900,
-            height=400,
+            width=600,
```

Comment:
The standard width is 640px on other plots. Did you need to reduce it in this plot?

Reply:
I will correct this.